generators/eslint/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+var generators = require('yeoman-generator');
+var extend = require('deep-extend');
+
+module.exports = generators.Base.extend({
+  constructor: function () {
+    generators.Base.apply(this, arguments);
+
+    this.option('babel', {
+      required: false,
+      defaults: false,
+      desc: 'Compile ES2015 using Babel'
```

Comment:
Done acdb8172318207d790bc1785d69eabb7ad45f12e

Should I rename babel option in whole generator ?